docs/dev-guide/design/go-migration.md

```
@@ -0,0 +1,176 @@
+        Skill and Group commands                :2026-07-15, 3w
+    section Release
+        GoReleaser setup and first release      :2026-08-01, 1w
+        Freeze Python (v0.x-python-final)       :2026-08-01, 1w
+        Remove Python source                    :2026-08-15, 1w
+```
+
+## Coexistence Layout
+
+During the transition, both Python and Go source live in the same repository:
+
+```
```

markdownlint-cli2 (0.22.1)

[warning] Fenced code blocks should have a language specified

(MD040, fenced-code-language)